docs/0.3.5-alpha/by_example/ubuntu_updater.md

```diff
@@ -16,7 +14,7 @@ main {
   silent unsafe $ iwgetid -r | grep -E '(OEBB|WESTlan)' $
   if status == 0 {
     echo "Skipping updates because of slow Wifi"
-    exit(0)
+    exit 0
```

Comment:
Wasn't `exit` still in the standard library in 0.3.5?

Reply:
I don't think that was in 0.3.5

Reply:
Ah you're right @hdwalters @Mte90 I thought that it was already a part of 0.3.5. Double checked on that and it seems that it was not. I'll fix this